plugins/catalog-backend-module-incremental-ingestion/src/types.ts

```diff
@@ -154,11 +167,11 @@ export interface IterationEngine {
   taskFn: TaskFunction;
 }
 
-export interface IterationEngineOptions {
+export interface IterationEngineOptions<TInput> {
```

Comment:
Same here, should just leave out the type argument, and having `unknown` below

Reply:
Done!